readme.md

```diff
@@ -0,0 +1,74 @@
+## Ciris AWS SSM
+
+[Amazon EC2 Systems Manager](https://aws.amazon.com/ec2/systems-manager/?nc2=h_m1) (also known as Simple Systems Manager or SSM) support for [Ciris](https://cir.is).
+
+Enables us to load parameters of type "string" or "secret string" from the SSM parameter store.
```

Comment:
While you're changing stuff, you could correct this. Everywhere I've written "secret string", the correct AWS terminology is actually "secure string".

Reply:
This was the only occurrence. Fixed.